src/Appointments/App/Controllers/DeleteAppointmentController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Appointments\App\Controllers;
+
+use Illuminate\Http\Response;
+use Lightit\Appointments\Domain\Models\Appointment;
+
+final class DeleteAppointmentController
+{
+    public function __invoke(Appointment $appointment): Response
+    {
+        $appointment->delete();
+
+        return response()->noContent();
```

Comment:
This is something i discussed with sergio and in order to always return a JsonResponse return this: 
```suggestion
        return response()->json(null, JsonResponse::HTTP_NO_CONTENT);
```